tutorials/esp32_blogpost.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Hello World- ESP32 Firmware
+layout: post
+created_at: Wed Aug 29 2018
+permalink: tutorials/esp32_blogpost
+current: blog
+author: Arushi
+twitter: arushi1803
+categories:
+- student-posts
+---
+
+We’re currently aiming to make our contribution to the Tessel project. Tessel-Reach is a developing concept that we 
+recently got introduced to.<br>
+It aims at creating a star topology network with a Tessel 2 board as the hub and Reach modules as low power nodes, 
+that are currently being configured on an ESP32 Thing.<br>
+The latest development on this project was to push a hello world program onto the firmware of ESP32. We began 
+by achieving the same, while following the ESP-IDF tutorial, in order to familiarise ourselves with the 
+platform we would be working on.<br>
+Connecting the ESP32 module to the laptop was just about plugging it into a USB port via a micro-USB 
+cable. But that is not enough to run the module on Windows 10. We need to setup a toolchain so that we can ‘talk’
+with the sensor i.e. push code to the sensor and make it do some exciting stuff :D<br>
+Here’s how we went about it...<br>
+## Step 1
+We kick-started the process by setting up the toolchain for the ESP32 module. This took about an hour — first 
+downloading, then unzipping the compressed file. The toolchain provided an environment to interact with the ESP32 
+module on Windows 10 through minGW32 CLI. IDF_PATH was set as the system environment variable pointing to to the 
+‘esp-idf’ folder in esp.<br>
+(/img/esp1.png)
+## Step 2
+The hello world folder loaded into the msys32 folder. We had to the set IDF_PATH correctly. It was initially set 
+to the esp folder but it worked only with the esp-idf folder. The path given on the website did not work.<br>
+(/img/esp2.png)
+## Step 3
+We then gave the make menuconfig command and selected the serial flash option. Following that, we selected the default
```

Comment:
Can you add quotes ("") or backticks (``) around the `make` commands so they are easier to read?